src/ecs/commands/runCommandInContainer.ts

```diff
@@ -121,11 +118,24 @@ export async function runCommandInContainer(
         if (!command) {
             return
         }
+        let ssmPlugin: string | undefined
+        if (!settings.readDevSetting<boolean>('aws.dev.forceInstallTools', 'boolean', true)) {
```

Comment:
Nit/TODO:
May want to move reading the developer setting + installing + getting the CLI path into a single shared function for all CLI tools. Seems like it'd be a common pattern to me. Something like `ensureCli`. Can just let the error bubble up, failed dependency is good information to surface.